### ISSUE TITLE
electric motor not working

### ISSUE THREAD

**Issue by kiven7e** [mrh0/createaddition#329]
**Describe the bug**
electric motor is not working

**To Reproduce**
generate power with alternator (tested that it worked with a tech reborn MFE)
use the wires and connectors (also tested it with tech reborn cables)
nothing happens

**Additional context**
modpack is all of fabric 5 where i added the create addition mod
when i was testing it with a alternator connected to the output of thew motor to add stress my game crashed full log here [https://mclo.gs/aMILOby](https://mclo.gs/aMILOby)
```
[16:54:16] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking block entity
```

**Comment by Nerodon** [comment]
In create 5.0c with the latest Create Additions, it works fine.
This seems to happen on Fabric when updating to Create 5.0d

Loading any world with electric motors on Create 5.0d will crash the client

`[04:10:56] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking block entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:911) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:851) ~[client-intermediary.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:97) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:273) ~[client-intermediary.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.NoSuchMethodError: 'net.minecraft.class_2588 com.simibubi.create.foundation.utility.Lang.translateDirect(java.lang.String, java.lang.Object[])'
	at com.mrh0.createaddition.blocks.electric_motor.ElectricMotorTileEntity.addBehaviours(ElectricMotorTileEntity.java:75) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.simibubi.create.foundation.tileEntity.SmartTileEntity.<init>(SmartTileEntity.java:43) ~[create-fabric-1.18.2-0.5.0d-730.jar:?]
	at com.simibubi.create.content.contraptions.base.KineticTileEntity.<init>(KineticTileEntity.java:72) ~[create-fabric-1.18.2-0.5.0d-730.jar:?]
	at com.simibubi.create.content.contraptions.base.GeneratingKineticTileEntity.<init>(GeneratingKineticTileEntity.java:23) ~[create-fabric-1.18.2-0.5.0d-730.jar:?]
	at com.mrh0.createaddition.blocks.electric_motor.ElectricMotorTileEntity.<init>(ElectricMotorTileEntity.java:59) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.tterrag.registrate.builders.BlockEntityBuilder.lambda$createEntry$2(BlockEntityBuilder.java:132) ~[registrate-fabric-MC1.18.2-1.1.4-3d1384eafe34fc4e.jar:?]
	at net.minecraft.class_2591.method_11032(class_2591.java:84) ~[client-intermediary.jar:?]
	at com.tterrag.registrate.util.entry.BlockEntityEntry.create(BlockEntityEntry.java:28) ~[registrate-fabric-MC1.18.2-1.1.4-3d1384eafe34fc4e.jar:?]
	at com.mrh0.createaddition.blocks.electric_motor.ElectricMotorBlock.method_10123(ElectricMotorBlock.java:62) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at net.minecraft.class_2818.method_12208(class_2818.java:292) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818.method_12201(class_2818.java:315) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_8321(class_1937.java:533) ~[client-intermediary.jar:?]
	at com.mrh0.createaddition.energy.BaseElectricTileEntity.updateCache(BaseElectricTileEntity.java:104) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.mrh0.createaddition.energy.BaseElectricTileEntity.firstTick(BaseElectricTileEntity.java:97) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.mrh0.createaddition.energy.BaseElectricTileEntity.tick(BaseElectricTileEntity.java:92) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.mrh0.createaddition.blocks.connector.ConnectorTileEntity.tick(ConnectorTileEntity.java:208) ~[createaddition+1.18.2-1.18.2-20220813a.jar:?]
	at com.simibubi.create.foundation.tileEntity.SmartTileEntityTicker.tick(SmartTileEntityTicker.java:15) ~[create-fabric-1.18.2-0.5.0d-730.jar:?]
	at net.minecraft.class_2818$class_5563.method_31703(class_2818.java:659) ~[client-intermediary.jar:?]
	at net.minecraft.class_2818$class_5564.method_31703(class_2818.java:713) ~[client-intermediary.jar:?]
	at net.minecraft.class_1937.method_18471(class_1937.java:475) ~[client-intermediary.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:368) ~[client-intermediary.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:907) ~[client-intermediary.jar:?]`

**Comment by mrh0** [comment]
fixed in 20220911a.
